### ISSUE TITLE
sales_order.search() says invalid api path when used for magento 1.9

### ISSUE THREAD

**Issue by Raj160907** [fulfilio/python-magento#26]
## Expected Behavior
client.order.search() -> should give the function takes5 arguments but one provided

## Actual Behavior
client.order.search() -> says invalid api path

## Steps to Reproduce the Problem

  1.
  1.
  1.

## Specifications

  - Version: latest
  - Platform: magento 1.9
  - Subsystem:

**Comment by Gabriel712** [comment]
Hello, 

I tested here, the same problem.

-> Solution get all
client.order.list()
And use your self a filter
--

**Comment by Raj160907** [comment]
but in client.order.list() -> *pagination *doesn't seem to work.

**Comment by Raj160907** [comment]
pagination is not supported in sales_order.list() API

**Comment by Gabriel712** [comment]
Hi, i tested this here. It's OK!
How many products you have ?

**Comment by Raj160907** [comment]
i dont have products but i have 100 orders and i want it to display 25 orders per page

**Comment by Gabriel712** [comment]
*pagination *doesn't seem to work. * is your error ?

**Comment by Raj160907** [comment]
yes, please help me how can i apply pagination in order.list() or in any api?

**Comment by Gabriel712** [comment]
show me your code

**Comment by Raj160907** [comment]
sales_order.search() says invalid api path and there's nothing like pagination in sales_order.list() api

**Comment by Gabriel712** [comment]
Maybe client.order.search() have a error;

Use the client.order.list()

**Comment by Raj160907** [comment]
please tel me how to use client.order.list() for pagination

**Comment by Gabriel712** [comment]
Raj, i found a error. 

"invalid api"
is an error of importation, 
Call me, frix131@gmail.com 

http://devdocs.mageshop.com.br/devdocs.magento.com/guides/m1x/api/soap/sales/sales.html
